### PR TITLE
Updating Release Notes for Release/7.x

### PR DESCRIPTION
**Pull request by kkeirstead**
I think I got everything here - if I missed something, please let me know!

### CODE REVIEW

File: documentation/releaseNotes/releaseNotes.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 Today we are releasing the next preview of the `dotnet monitor` tool. This release includes:
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.md:1]:
```suggestion
Today we are releasing the next official preview of the `dotnet monitor` tool. This release includes:
```

File: documentation/releaseNotes/releaseNotes.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
 
 - Load a profiler via a collection rule action (#781)
-- Get or Set an environment via a collection rule action (#1176)
+- Get or Set an environment variable via a collection rule action (#1176)
 - Add `MachineJson` option to `--output` parameter for `generatekey` command. This is useful for automating credential provisioning. (#1428)
+- Allow for simplified diagnostic port configuration (#1292)
+- Ensure `config show` displays arrays without indices (#1054)
+- ⚠️Remove netcoreapp3.1 TFM (#1267)
+- Add latest .NET 7.0 SDK (#1268)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.md:9]:
```suggestion
- Add `net7.0` TFM #1269
```

File: documentation/releaseNotes/releaseNotes.md [hunk 3]
```diff
@@ -1,7 +1,11 @@
 Today we are releasing the next preview of the `dotnet monitor` tool. This release includes:
 
 - Load a profiler via a collection rule action (#781)
-- Get or Set an environment via a collection rule action (#1176)
+- Get or Set an environment variable via a collection rule action (#1176)
 - Add `MachineJson` option to `--output` parameter for `generatekey` command. This is useful for automating credential provisioning. (#1428)
+- Allow for simplified diagnostic port configuration (#1292)
+- Ensure `config show` displays arrays without indices (#1054)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.md:7]:
```suggestion
- Ensure `config show` command displays arrays without indices (#1054)
```

**Review by jander-msft — CHANGES_REQUESTED** [review]

File: documentation/releaseNotes/releaseNotes.md (hunk 2 above)
Comment (kkeirstead) [reply documentation/releaseNotes/releaseNotes.md]: I figured you meant for the parentheses to be there so I manually made the change; I also added the tick marks around `netcoreapp3.1` like you did for `net7.0`

**Review by jander-msft — APPROVED** [review]

**Pull request merged**